classes/TwitterClient.php

```diff
@@ -95,8 +97,10 @@ public function getLatestTweet($params)
             $text = preg_replace('/\#\w+/', '<span class="tag">$0</span>', $text);
             $message['text_processed'] = $obj->urlsToLinks($text);
         }
-
-        Cache::put($cacheKey, serialize($result), 2);
+		
+		$expiresAt = Carbon::now()->addMinutes($params["cache-duration"]);
```

Comment:
whitespace

Reply:
Also why are you using Carbon here instead of just passing the cache-duration directly?